src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -17,6 +17,7 @@ object ConsignmentFields {
   case class ChecksumProgress(filesProcessed: Int)
   case class FFIDProgress(filesProcessed: Int)
   case class FileChecks(antivirusProgress: AntivirusProgress, checksumProgress: ChecksumProgress, ffidProgress: FFIDProgress)
+  case class ParentFolder(parentFolder: Option[String] = None)
```

Comment:
Oh yeah, I hadn't thought about that. I initially had it nested because I created a new API endpoint, then didn't think to undo that when I changed it to be part of the deferred resolver, etc. I'll try to un-nest so it's a less complex data structure :+1: